<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/img/_bkPage.jpg);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white;</string>
       </property>
       <property name="text">
        <string>客户端状态：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="clientListWidget">
       <property name="font">
        <font>
         <family>华文隶书</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">selection-background-color: darkgray;
color: white;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white;</string>
       </property>
       <property name="text">
        <string>消息文本：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit_2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>李旭科漫画体v1.0</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid gray;/*设置边框的粗细，以及颜色*/
border-radius: 10px;/*设置圆角的大小*/
padding: 0 8px;/*如果没有内容光标在开始往后移动0.8像素点*/
selection-background-color: darkgray;
color: white;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="InfoPushButton">
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: white;</string>
         </property>
         <property name="text">
          <string>发送消息</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="DelInfoPB">
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: white;</string>
         </property>
         <property name="text">
          <string>清空消息</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white;</string>
       </property>
       <property name="text">
        <string>运行日志:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit">
       <property name="font">
        <font>
         <family>李旭科漫画体v1.0</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid gray;/*设置边框的粗细，以及颜色*/
border-radius: 10px;/*设置圆角的大小*/
padding: 0 8px;/*如果没有内容光标在开始往后移动0.8像素点*/
selection-background-color: darkgray;
color: white;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white;</string>
       </property>
       <property name="text">
        <string>消息对话：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="minimumSize">
        <size>
         <width>330</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>330</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>素晴字体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid gray;/*设置边框的粗细，以及颜色*/
border-radius: 10px;/*设置圆角的大小*/
padding: 0 8px;/*如果没有内容光标在开始往后移动0.8像素点*/
selection-background-color: darkgray;

QListWidget{background-color:  rgb(247, 247, 247);  border: 1px solid  rgb(247, 247, 247);outline:0px;}
QListWidget::Item{background-color: rgb(247, 247, 247);}
QListWidget::Item:hover{background-color: rgb(247, 247, 247); }
QListWidget::item:selected{
	background-color: rgb(247, 247, 247);
	color:black; 
    border: 1px solid  rgb(247, 247, 247);
}
QListWidget::item:selected:!active{border: 1px solid  rgb(247, 247, 247); background-color: rgb(247, 247, 247); color:rgb(247, 247, 247); } </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="rizhisave">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white;</string>
       </property>
       <property name="text">
        <string>保存日志</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="riziclear">
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white;</string>
       </property>
       <property name="text">
        <string>清空日志</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
